code/frameworks/angular/src/server/plugins/storybook-normalize-angular-entry-plugin.js

```diff
@@ -25,23 +25,28 @@ export default class StorybookNormalizeAngularEntryPlugin {
   }
 
   apply(compiler) {
-    const webpackOptions = compiler.options;
-    const entry =
-      typeof webpackOptions.entry === 'function' ? webpackOptions.entry() : webpackOptions.entry;
+    compiler.hooks.environment.tap(PLUGIN_NAME, () => {
```

Comment:
This wrapper is the only addition. The rest of the changes are formatting related.

Reply:
The change is also backwards compatible. Tested it with Angular 15.0.3. Angular 15.0.4 introduced the issue.